### PR TITLE
Add the allergies exercise

### PR DESCRIPTION
**Pull request by DarkoGNU**
I hope everything's gone well
I'm open to suggestions

### CODE REVIEW

**Comment by DarkoGNU** [comment]
Oh, sorry for that error

**Comment by DarkoGNU** [comment]
I've just squashed that commit which repaired the notebook with the other commit. I hope I've done the right thing

File: exercises/allergies/allergies.jl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function allergic_to(score, allergen)
+
+end
+
+function list(score)
```
Comment (SaschaMann) [inline exercises/allergies/allergies.jl]: Maybe `allergy_list` would be a better name. List is quite generic.

**Comment by SaschaMann** [comment]
Thanks for adding the exercise!

I think the name of `list` should be changed but everything else looks good to me.

> I've just squashed that commit which repaired the notebook with the other commit. I hope I've done the right thing

Yep, you did. Thanks!

**Comment by DarkoGNU** [comment]
Is there anything left to do?

**Comment by SaschaMann** [comment]
> Is there anything left to do?

Nope, looks nice! Thanks again for adding the exercise :)

@logankilpatrick Previously `allergy_list` was called `list`

**Comment by DarkoGNU** [comment]
> Looks good to me. @SaschaMann what do you mean by the List?

@logankilpatrick one question - is force push a good way to make changes to a pull request like this? I'm new to Git and I see that maybe your confusion above wouldn't have arosen if I had done it another way

**Comment by logankilpatrick** [comment]
PM me on slack and we can talk about it!

**Comment by SaschaMann** [comment]
Ideally you should wait with squashing and forcepushing until the reviewing process is complete or until the maintainers ask for it. It's easier to do reviews on short commits a preserved complete dev history, even when the PR will be squashed before or on merging it to master.

And in general, try to use `--force-with-lease` instead of `--force` when possible (it has some safety features) and *never* force push branches that others work on (not the case for most PRs, though).

**Comment by DarkoGNU** [comment]
Understood, thanks!

**Pull request merged**